src/main/resources/templates/admin.html

```diff
@@ -307,6 +310,15 @@ <h2 class="mt-md-1 mt-0 d-inline-block">Manage Person</h2>
                         </div>
                     </div>
                 </div>
+                <div th:if="${!@realm.isProduction()}" class="tab-pane fade" id="throw-exception" role="tabpanel">
+                    <div class="row m-auto pt-3 pb-3 d-flex">
+                        <div class="col-lg-9 col-md-8 col-12 p-0 text-center text-md-left">
+                            <h2 class="mt-md-1 mt-0 d-inline-block">Throw Exception</h2>
+                        </div>
+                    </div>
+                    <div class="alert alert-warning" th:text="#{screen.message.admin.page.exception.tab.description}"></div><br>
+                    <button class="btn btn-primary" role="button" ng-click="throwException()">Throw Exception</button>
+                </div>
```

Comment:
I'm thinking we should rename this new tab from "Throw Exception" to something general for testing so that in Hokwai and Jarren's [ticket](https://uhawaii.atlassian.net/browse/GROUPINGS-1475), they can also put their buttons that runs their commands here too. Then we will have one centralized place to run certain tests on the Admin page.